platform/mellanox/mlnx-platform-api/sonic_platform/module.py

```diff
@@ -267,7 +267,7 @@ def __init__(self, dpu_id):
         self.CONFIG_DB_NAME = "CONFIG_DB"
         self.DHCP_SERVER_HASH = f"DHCP_SERVER_IPV4_PORT|bridge-midplane|{self._name.lower()}"
         self.DHCP_IP_ADDRESS_KEY = "ips@"
-        self.config_db = ConfigDBConnector(use_unix_socket_path=True)
+        self.config_db = ConfigDBConnector(use_unix_socket_path=False)
```

Comment:
@prgeor Do we need this change in other SKUs?

Reply:
no